typescript/packages/well-log-viewer/src/components/WellLogView.tsx

```diff
@@ -1309,21 +1311,21 @@ class WellLogView
         // set callback to component's caller
         this.props.onCreateController?.(this);
 
-        this.setControllerZoom();
-
         this._isMount = false;
     }
 
     componentDidMount(): void {
+        if (this._isMount) return; // anti-StrictMode workaround
```

Comment:
Why do we need to do that ?

StrictMode is dev only, not release...

Reply:
There was a some wrong behaviour which was discovered with StrictMode.
I rewrite the code a little to remove the **if**